site/en/docs/extensions/mv3/user_interface/index.md

```diff
@@ -35,51 +60,50 @@ The `"action"` field is registered in the manifest.
 }
 ```
 
-Declaring `"action"` keeps the icon colorized, indicating the extension is available to
-users.
+###  Activate the action conditionally  {: #activate_pages }
 
-### Add a badge {: #badge }
+The [declarativeContent][api-declarativecontent] API allows you to enable and disable the action
+based on the page URL or if the CSS selectors match the elements on the page.
 
-Badges display a colored banner with up to four characters on top of the browser icon. They can only
-be used by extensions that declare `"action"` in their manifest.
+When an extension is disabled, the icon is greyscaled. If the user clicks on the disabled extension,
+the extension's context menu will appear.
 
-Use badges to indicate the state of the extension. The [Drink Water Event][2] sample displays a
-badge with "ON" to show the user they successfully set an alarm and displays nothing when the
-extension is idle.
+<figure>
+{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/hlYsQJPFsF7WBAjJZ6DS.png", 
+alt="Clicked Disabled extension", width="252", height="180", class="screenshot"%}  
 
-{% Img src="image/BrQidfK9jaQyIHwdw91aVpkPiib2/nXwAHSWLBEgT8099ITT0.png",
-       alt="Badge On", height="72", width="72" %}
+<figcaption>
+    Disabled extension.
+  </figcaption>
+</figure>
 
-{% Img src="image/BrQidfK9jaQyIHwdw91aVpkPiib2/pNz8UgfTBMmcf7fE9wja.png",
-       alt="Badge Off", height="72", width="72" %}
+See [Emulating pageActions with declarativeContent][docs-emulating-page-actions] for an example on
+how to disable the action by default and use the declarativeContent API to enable the action on
+specific sites.
```

Comment:
Might be good to set some context on what a pageAction is before directing the reader to guidance on how to emulate them.

Reply:
I think I'm going to remove it for now.